x/did/types/errors.go

```diff
@@ -0,0 +1,30 @@
+package types
+
+// DONTCOVER
+
+import (
+	"fmt"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// x/did module sentinel errors
+var (
+	ErrDIDExists = sdkerrors.Register(ModuleName, 101, "DID %v already exists")
```

Comment:
```suggestion
	ErrDIDExists = sdkerrors.Register(ModuleName, 1, "DID %v already exists")
```
Let's use error codes starting from `1`. I'm not sure why it was starting from `101` previously, but it seems it's safe to use `1`~.

Reply:
@youngjoon-lee 
OK. I'll fixed it. 
Start code number 1